LICENSE

```diff
@@ -0,0 +1,46 @@
+Blu's License of Common Sense
+====================================
+
+Here's how it goes: I like making my code visible so that others may learn from it. Reason for that is because I myself learned a lot that way.
+What I don't like is anyone's creative property being stolen, used without credit or commercialized without permission.
+
+As such, this is my custom license which should hopefully cover all that:
+
+#### What you *can* do:
+- Look at the code, draw inspiration, copy bits of it to use in your own projects
+- Fork this repository and modify its code
+- Submit Pull Requests to this repository
+- Write other projects that base on this project and its API (if present) (This means Addons are totally cool)
+
+#### What you *can not* do:
+- Claim any of this code to be your property
+- Redistribute this project (source or compiled) without explicit permission of the license holder, BluSunrize
+- Copy large sections of this code (classes, complex methods) without giving credit
+- Monetize any of this project. This means selling the code, compiled projects and also forbids the use of paywalls or "donations" to restrict access to any part of the project's content. I specifically exempt CurseForge's "Points"-system from this statement.
+
+#### What you *have to* abide by:
+- If you have used any code from this project (with the exception of API classes), that project must also be visible source. This allows others to learn from your work.
+- Give proper credit where it's due when copying copy (see above)
```

Comment:
`when copying copy` - This part reads a little weirdly, maybe you meant `when copying code`?

Reply:
I agree, `when copying copy` makes no sense. One way to resolve this is to `...when making copies` or simply `when copying portions of the code`

Reply:
should have been "when copying code" will fix =P